src/analysis/rust_type.rs

```diff
@@ -285,7 +286,7 @@ pub fn rust_type_full(
                             } else {
                                 "&"
                             },
-                            if y != "GString" {
+                            if !y.ends_with("GString") {
```

Comment:
This could give false positives

Reply:
Do you one in mind? I could rewrite it like this:

```rust
if y != "GString" && !y.ends_with("::GString") {
```

Reply:
Works for me, but do it consistently everywhere please

Reply:
Ok, new helper function incoming!

Reply:
I like helper functions